tools/apigee-sackmesser/bin/sackmesser

```diff
@@ -158,6 +165,7 @@ if [[ -z "$token" && "$apiversion" = "apigee" ]]; then
       --data "grant_type=password")
     token=$(echo "$response" | jq -r '.access_token')
     export token
+    echo "token from sackmesser: $token"
```

Comment:
@SamPriyadarshi Let's remove printing of the Token.

Reply:
Remove this echo statement.